### DISCUSSION TITLE
How to detect special symbol

### DISCUSSION THREAD

**Discussion by giri-ggr** [jsvine/pdfplumber#812] [Q&A]
extract_text and extract_tables not able to detect bullet point symbol before the text and is also not image, can someone pls give me idea, how to detect it.

**Comment by samkit-jain** [comment]
Hi @giri-ggr Appreciate your interest in the library. They could be curves or rects. You can check using `page.curves` and `page.rects`. For better visual identification, you can use the [drawing methods](https://github.com/jsvine/pdfplumber#drawing-methods). Also, to assist you further, please share the PDF redacting any sensitive information.
